lib/omnibus/packagers/deb.rb

```diff
@@ -673,5 +675,35 @@ def install(packages, enablerepo = NULL)
     def remove(packages)
       shellout!("apt-get remove -y --force-yes #{packages}")
     end
+
+    def compression_level(val = nil)
+      unless val.nil?
+        unless val >= 0 && val <= 9
+          raise InvalidValue.new(:compression_level, 'be an Integer between 0 and 9 included')
+        end
+      end
+      @compression_level = val || 6
+    end
+    expose :compression_level
+
+    def compression_threads(val = nil)
+      unless val.nil?
+        unless val > 0 && val < 32
+          raise InvalidValue.new(:compression_threads, 'be a stricly positive and lower than 32 Integer')
+        end
+      end
+      @compression_threads || 1
```

Comment:
Is it working like that? Or should it be done like you do with `compression_level`
```
@compression_level = val || 6
```
If both work maybe you could use the same syntax everywhere

Reply:
It doesn't work, and I forgot to fix it in a few places, thanks for spotting!